src/components/incidents/IncidentsCard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { DateTime } from 'luxon';
+import { Card } from 'antd';
+import 'antd/dist/antd.css';
+
+const IncidentsCard = props => {
+  let cityState = `${props.incident.city}, ${props.incident.state}`;
+
+  return (
+    <div className="cardBox">
+      {/* for states, I was thinking we could import the abbreviations from the bargraphAssets file */}
```

Comment:
This kind of comment is good to include in PR or PR draft for discussion. I'd limit these comments in the future.

Reply:
I will make sure to include those kind of comments in the PR next time, thank you.